### ISSUE TITLE
CMS sitewide 508-defect-2 [SEMANTIC MARKUP]: Heading levels SHOULD only increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12315]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team affected: VSA-BAM2; Project: Debt Letters MVP
```
Already logged: https://github.com/department-of-veterans-affairs/va.gov-team/issues/7708

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Headings **should** increase by one. 

When the alert box component was brought over from Formation it was brought over with an h3 as the heading. In Formation, there is a `level` prop that may be set to ensure the proper heading levels. For the CMS, another approach should be taken to design these as static components to ensure heading levels increase by one, so that screen reader users understand the semantic hierarchy of the page.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand the content hierarchy of the page.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Log in as user 1
1. Enter https://staging.va.gov/health-care/order-hearing-aid-batteries-and-accessories/?postLogin=true in browser
1. Run an axe browser scan
1. Verify the "Heading levels should only increase by one" issue appears, related to `#am-i-eligible-to-order-hearing`

## Possible Fixes (optional)

For the CMS alert boxes, the recommendation is the h3 become an h2 with the h3 utility class.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)
* [MDN dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)
* [MDN ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role)
* [MDN Description List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

## Screenshots or Trace Logs

![Screen Shot 2020-08-10 at 3 52 04 PM](https://user-images.githubusercontent.com/57469/89824662-7a457f00-db21-11ea-8c78-956fd1785d17.png)

**Comment by jenstrickland** [comment]
Apologies, got the urls for products mixed up trying to access the debt letters product.
